lib/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.24)
 
-project(streaming_protocol VERSION 0.10.5 LANGUAGES CXX)
```

Comment:
We should not have this duplication. I'll give you a PR afterwards to have it only once.